### PR TITLE
Create security-privacy-questionnaire.md

### PR DESCRIPTION
**Pull request by mounirlamouri**
@bialpio PTAL

### CODE REVIEW

File: security-privacy-questionnaire.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+This document answers the [W3C Security and Privacy
+Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/) for the
+WebXR Hit Test Module specification.
+
+**What information might this feature expose to Web sites or other parties,
+and for what purposes is that exposure necessary?**
+
+This feature will help a web site to map the user environment. Hit tests, in the
+context of VR will only help map the 3D environment. However, in the context of
```
Comment (bialpio) [inline security-privacy-questionnaire.md]: I think that depending on the exact implementation, the VR headsets could also potentially map the real world around the user. The initial implementations would probably leverage the headsets' information of the 3D play area, but it seems to me that spec-compliant implementation on VR could do the same thing that AR implementation is doing.

File: security-privacy-questionnaire.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+The specification allows a user agent to restrict the usage of hit tests
+subscriptions based on a XRRay. Using a lot of rays would allow the web site to
+have a more detailed view of the environment and a user agent may decide that
+after a certain number, the requests are being superfluous.
+
+This feature of WebXR has to be listed when requesting an XR session which
+allows the user agent to show a prompt specifically requesting user approval.
+
+However, it's worth noting that a WebXR module will expose planes in the user
```
Comment (bialpio) [inline security-privacy-questionnaire.md]: Nit: s/"a WebXR module"/"a planned WebXR module".

File: security-privacy-questionnaire.md (hunk 1 above)
Comment (mounirlamouri) [reply security-privacy-questionnaire.md]: I guess VR headsets could expose play areas and stuff like that and expose it with hit tests. Would you like me to mention this? (Running to a meeting so I can do that afterwards.)

File: security-privacy-questionnaire.md (hunk 2 above)
Comment (mounirlamouri) [reply security-privacy-questionnaire.md]: Done.

File: security-privacy-questionnaire.md (hunk 1 above)
Comment (mounirlamouri) [reply security-privacy-questionnaire.md]: Applied comment by dropping AR/VR distinction.

**Pull request merged**